### ISSUE TITLE
Migrate staging to 18f.gov domain

### ISSUE THREAD

**Issue by adelevie** [18F/myusa#625]
Within a matter of weeks, we will need to stop using the 18f.us domain. To give our staging users the maximum amount of time to update their applications to the new URL (myusa-staging.18f.gov), we should point both domains to our staging ELB.

This ticket is a dependency for the MyUSA ATO. For the purpose of the ATO the follow tasks need to occur: 
- [x] Make sure no one is spinning up new instances that point to 18f.us myusa staging, since that is being decommissioned. 
- [x] Notify points of contact for all staging integrators that a change is on the horizon.
- [x] Notify integrators that they may now update their applications

The rest of the work associated with this ticket, which is not ATO dependent is in #687. 

cc @yozlet @dlapiduz

**Comment by NoahKunin** [comment]
Hey team - trying to put a wrap on all the relates tasks, but seeing this has been pushed out over 4 sprints.

Hoping maybe the checklist isn't updated @yozlet and we're closer to the final step?

**Comment by yozlet** [comment]
We have https://staging.my.usa.gov/ up and running with migrated data from old AWS staging. @jackiekazil is emailing all integrators and telling them to use that new address.

@NoahKunin, do we still need to repoint myusa-staging.18f.gov at the ELB? (We currently have an ELB for staging.my.usa.gov) Right now it's CF-native.

**Comment by jackiekazil** [comment]
@yozlet, et. al -- integrators have been notified; I cc-ed you on the email. Now, I will do the case down as needed.

**Comment by NoahKunin** [comment]
For the sake of the ATO, I can live with the idea that we aren't creating any more *.18f.us endpoints or whatever. If / once that's the case, we're good, and then 18f.us can fully deprecate over the first 90 days.

cc to @jackiekazil to split into another issue or what have you.

**Comment by jackiekazil** [comment]
@18F/myusa, @dlapiduz -- I updated this ticket scope after a convo w/ @NoahKunin about what should happen before ATO and what can happen after. The items that can move beyond ATO have been moved to #687, so we can close this.
